### ISSUE TITLE
🐞 - If animation is disabled, then postfix prefixes don't work in `inputNumber`

### ISSUE THREAD

**Issue by splincode** [Tinkoff/taiga-ui#3823]
### Which @taiga-ui/* package(s) are the source of the bug?

kit

### Please provide a link to a minimal reproduction of the bug

https://taiga-ui.dev/stackblitz

### Is this issue blocking you?

Non-Blocking

### Description

```css
* {
  animation: none !important;
  transition: none !important;
}
```

@waterplea @nsbarsukov @vladimirpotekhin what do you think is it expected behavior?

<img width="560" alt="image" src="https://user-images.githubusercontent.com/12021443/223699362-0b0d37ef-4b2c-429b-b547-9fe0f3c3a8f3.png">
<img width="359" alt="image" src="https://user-images.githubusercontent.com/12021443/223699384-445d15b0-3af2-4c33-8871-f9eae9e30b3f.png">

### Angular version

latest

### Taiga UI version

latest

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

**Comment by splincode** [comment]
if I comment current line

<img width="697" alt="image" src="https://user-images.githubusercontent.com/12021443/223705950-6960583f-b84d-4acb-aead-97c4b8b3a20f.png">

then works as expected

<img width="680" alt="image" src="https://user-images.githubusercontent.com/12021443/223705990-4b875b32-0d36-4417-9cbf-60dcfa50619c.png">

**Comment by waterplea** [comment]
I don't think you can rely on everything working properly, if you enforce your custom styles on everything.
